code/_onclick/item_attack.dm

```diff
@@ -1,79 +1,80 @@
-

```

Comment:
Why does github want to change the whole file? In any case, only lines 15-21 are changed